src/components/PageSections/Menu/_FullSiteMenu.scss

```diff
@@ -101,13 +130,13 @@
     width: 25%;
     padding: $coa-spacing-large $coa-spacing-small $coa-spacing-large 0;
 
-    &:nth-child(n + 5) {
+    &:nth-child(-n+4) {
```

Comment:
Not gonna mark it as blocking, but for maintainability it would be great to at least have a comment about what this element is doing, and ideally refactor it to be more legible.